src/index.js

```diff
@@ -389,18 +416,26 @@ export default class TruncateMarkup extends React.Component {
       return string;
     }
 
+    if (this.policy.tokenizeString) {
+      return this.splitArray(
+        this.policy.tokenizeString(string),
+        splitDirections,
+        level,
+      );
```

Comment:
we need to `.join('')` after splitting otherwise the backwards compatibility breaks (because it changes the structure from a string on entry to an array on exit - React doesn't mind, but the custom ellipsis callback may have some issues with that :).

Reply:
ok